src/components/PopupWithImage.js

```diff
@@ -7,9 +7,9 @@ export default class PopUpWithImage extends Popup {
 
     super(popupSelector);
 
-    /* this._name = name;
+    this._name = name;
 
-    this._link = link; */
+    this._link = link;
```

Comment:
`{ name, link }` comes into `open` according to the task. Please, delete it from the constructor